### DISCUSSION TITLE
How to solve "There was a problem retrieving the support request details." error when check ticket details?

### DISCUSSION THREAD

**Discussion by AlexMisiulia** [microsoft/MicrosoftEdge-Extensions#98] [Discussions]
Hi! I create the [ticket](https://partner.microsoft.com/dashboard/v2/support/servicerequests/details?tid=2307240040007114) in Partner Center, but when I and my colleague check it, we have the error "There was a problem retrieving the support request details.". Maybe someone already solved this problem?

**Comment by PatriotPirate** [comment]
I hope that I didn't drop the ball ⚽

**Comment by Bhuvana-Priya** [comment]
Hi @AlexMisiulia,

We are looking into this and will get back to you with any further information.

Thank you

**Comment by caiiiycuk** [comment]
Hi @Bhuvana-Priy, @AlexMisiulia a, did you solved this issue? I faced exactly same problem in my support request [ticket](https://partner.microsoft.com/dashboard/v2/support/servicerequests/details?tid=2402150040001741)

**Reply by caiiiycuk** [reply]
@Bhuvana-Priya

**Reply by AlexMisiulia** [reply]
Hi @caiiiycuk. It's still unresolved. I wrote all people that I know and don't know in Linkedin, other social media and it didn't solve this issue.

**Comment by ManikanthMSFT** [comment]
Hey @AlexMisiulia, sorry for the inconvenience. Could you please confirm if the issue persists? If so, could you provide more details so we can assist you further?

**Reply by AlexMisiulia** [reply]
Hey @ManikanthMSFT. I still have the same issue and it blocks me from passing the "Account verification" so it's really CRITICAL right now. What details can I provide to solve this issue ASAP? Can I email you to speed up the process? Thank you!

**Reply by ManikanthMSFT** [reply]
Hi @AlexMisiulia, could you help us with your seller ID so that we can verify and assist you accordingly.

**Comment by aminesayah** [comment]
i have the same problem, any update?

**Reply by ManikanthMSFT** [reply]
Hi @aminesayah, could you help us with your seller ID so that we can verify and assist you accordingly.

**Comment by assoumemba** [comment]
Same issue, I got it this week. No way to see ticket details 

![image](https://github.com/user-attachments/assets/87a6a274-3ef6-41ed-8994-9c6a390670db)

**Reply by ManikanthMSFT** [reply]
Hi @assoumemba, could you help us with your seller ID so that we can verify and assist you accordingly.

**Reply by assoumemba** [reply]
Hi @ManikanthMSFT 
I don't know if you get my post, here is My Seller ID:54947690

Best regards

**Reply by ManikanthMSFT** [reply]
Hi @assoumemba, thank you for providing us with the seller5 ID. We are looking into this and will get back to you with more information.

**Comment by aminesayah** [comment]
yes , you can email me to speed the process

**Comment by assoumemba** [comment]
Hi
My Seller ID:54947690

Best regards.


Le lun. 3 févr. 2025 à 18:49, ManikanthMSFT ***@***.***> a
écrit :

**Reply by ManikanthMSFT** [reply]
HI @assoumemba, I hope this message finds you well.

We need to verify that your company owns the domain used during your enrollment into Partner Center. Please provide documentation that meets the following criteria:

Lists a domain that is an exact match to the one you are enrolling in Partner Center.
Includes your company's name, matching the company name used for enrollment.
Shows a purchase or renewal date not older than 12 months from the current date and not expiring in less than 60 days.
Acceptable forms of documentation include a Who Is lookup, an invoice, or a purchase receipt for the domain.

If it is not possible to provide additional documentation, please update your domain information to match the documents already provided.

Thank you for your cooperation. We look forward to your prompt response.

**Comment by aminesayah** [comment]
hello sorry for the late ,this is my   Seller: 81172430

Le jeu. 6 févr. 2025 à 09:27, ManikanthMSFT ***@***.***> a
écrit :

**Comment by aminesayah** [comment]
thanks for your support i think that the problem is fixed in platform, i
can now add payment and a tax

Le lun. 10 févr. 2025 à 12:44, ManikanthMSFT ***@***.***> a
écrit :

**Reply by assoumemba** [reply]
Thank you, the issue is now solved. 
Thanks for your support.

**Comment by yilinfalcon** [comment]
I faced the same problem, anyone can help? 
Seller: 90496070 
ticketsID:2503050010000763

![image](https://github.com/user-attachments/assets/e20c86e1-0159-40d1-8926-66542dcf0785)

**Reply by ManikanthMSFT** [reply]
Hi @yilinfalcon, we got an update on this, and our team is unable to verify your account based on the information you've provided and are requesting additional information.

We need to verify that your company owns the domain that you used when you enrolled into Partner Center. This can be a WhoIs lookup, or an invoice, purchase receipt for the domain, but it MUST meet the following criteria:

1) List a domain that is an exact match to the one you are enrolling in Partner Center.
2) Include your company's name (also match the company name you are enrolling into Partner Center)
3) Have a purchase, renewal date not older than 12 months from the current date and not expiring in less than 60 days.

If it is not possible to provide additional documentation, please update your domain information to match the documents already provided.

**Reply by yilinfalcon** [reply]
Thank you for your reply @ManikanthMSFT. 

 I have received an email from Microsoft support saying the same issue as you mentioned, and I replied that email with our company domain ownership certificate last Friday. But until now, our develop account is in the progress of employ verification and the status is still Rejected. And there is no further reply so far.

**Reply by ManikanthMSFT** [reply]
Hi @yilinfalcon, in the provided document address is not given to verify. Please provide more documents.

Acceptable document types include:
• Formation documents, such as articles of incorporation, partnership deed
• Franchise or agency appointment letters.
• Government issued letters, license, registration, or certificate.
• Lease or tenancy documents.
• Letter or statement from a financial institution or a utility company
• Record on a Government registry website (site/link must be displayed)
• Stock exchange filings or tax filing records.
• Certificate of incorporation or registration
• Extract from commercial register.
• Business license
• Tax certificate.
• DUNS certification

If applicable, documents submitted should be issued within the previous 12 months or where the expiration date is a future date that is at least 2 months away.

If it is not possible to provide additional documentation, please update your account information to match the documents already provided.

**Reply by ManikanthMSFT** [reply]
Hi @yilinfalcon, we appreciate your patience. We have received an update from our team stating that your account has been authorized. You can now verify and proceed further.

Thank you for your cooperation.

**Comment by novadonna** [comment]
I have the same exact issue I need to verify my account please as soon as possible my user name is support@novadonna.ae  and this is the case number 2503120040010938  will appreciate your support to verify my account please 
![Screenshot_20250313_013116_Chrome](https://github.com/user-attachments/assets/b4c4b253-b5b6-4628-94e7-1a1e0c277c3b)

@ManikanthMSFT

Thank you
Nevin

**Reply by novadonna** [reply]
I submitted the documents in this case number 2503120040010938  please support my case and verify my account please as soon as possible. 
Thank you so much for your support 
@ManikanthMSFT

**Comment by novadonna** [comment]
I submitted the documents in this case number 2503120040010938 please support my case and verify my account please as soon as possible.
Thank you so much for your support
@ManikanthMSFT

**Reply by novadonna** [reply]
my user name is support@novadonna.ae

**Reply by novadonna** [reply]
I'm trying to reach ti anyone to help me with my verification process as I can't access this ticket due to the same error 
So please will appreciate your support 
Thank you again so much 

![Screenshot_20250313_193620_Chrome](https://github.com/user-attachments/assets/26f0714c-4478-465e-874d-fc97a044ae43)

**Reply by novadonna** [reply]
@ManikanthMSFT

**Reply by novadonna** [reply]
I have provided all the required documents and still getting rejected my employment verification which is not making sense at all please help me with my issue
@ManikanthMSFT

**Reply by novadonna** [reply]
I really really need your support I have submitted all the required information now in this ticket number   2503130040014668 as I can't open previous tickets due to the same technical issue  please help me with my verification process. 
I need my developer account to be verified as soon as possible and I'm really tired from this lengthy process. 
Thank you so much for your support. 
@ManikanthMSFT

**Reply by ManikanthMSFT** [reply]
Hi @novadonna, we see that you have create a new query for the similar issue. Could we continue the discussion on the single channel to avoid confusion.

**Reply by novadonna** [reply]
Hi @ManikanthMSFT yes we continue the discussion In #285

**Comment by novadonna** [comment]
Thank you for your email, I'm attaching here the below documents in format pdf.On each attachment is mention:- My name: Nevin Omar Ali Mostafa Alkoly- My company name: Nova Donna FZC-Invoice of the Domain Renewal - novadonna.ae - 1 Year/s (23/06/2025 - 22/06/2026) dated on 03/03/2025 + Email ForwardingScreenshot of whois lookup in format pdf and if you are not able to open the attachment, It can be checked on this link,https://www.whois.com/whois/novadonna.aeorhttps://whois.aeda.net.ae/whois/whois_local.jsp?tab=0&t=1741279892414below is the result:Whois response for novadonna.ae:Domain Name    novadonna.aeRegistrar ID    TASJEELRegistrar Name    Host Arabia ( Tasjeel.ae )Status    okRegistrant Contact ID    DURRAQ-R63606JRegistrant Contact Name    Nevin Omar Ali Mostafa AlkolyRegistrant Contact Email   ***@***.*** Contact Organization    Nova Donna FZCTech Contact ID    DURRAQ-T63606JTech Contact Name    Nevin Omar Ali Mostafa AlkolyTech Contact Email   ***@***.*** Contact Organization    Nova Donna FZCName Server    ns5.tasjeel.aeName Server    ns6.tasjeel.aeAppreciate your support in verifying from your side .Best Regards,NevinSent from my Galaxy
-------- Original message --------From: ManikanthMSFT ***@***.***> Date: 17/03/2025  10:53  (GMT+04:00) To: microsoft/MicrosoftEdge-Extensions ***@***.***> Cc: novadonna ***@***.***>, Comment ***@***.***> Subject: Re: [microsoft/MicrosoftEdge-Extensions] How to solve "There was a
  problem retrieving the support request details." error when check ticket
  details? (Discussion #98) 
Hi @yilinfalcon, we got an update on this, and our team is unable to verify your account based on the information you've provided and are requesting additional information.
We need to verify that your company owns the domain that you used when you enrolled into Partner Center. This can be a WhoIs lookup, or an invoice, purchase receipt for the domain, but it MUST meet the following criteria:

List a domain that is an exact match to the one you are enrolling in Partner Center.
Include your company's name (also match the company name you are enrolling into Partner Center)
Have a purchase, renewal date not older than 12 months from the current date and not expiring in less than 60 days.

If it is not possible to provide additional documentation, please update your domain information to match the documents already provided.

**Comment by blaiz2023** [comment]
Same problem here.  Vetting process stuck at "Employment Verification", that is, until the vetting information vanished from my account.  Whatever that means.  My seller ID is: 90695370 and my email is blaizenterprises@gmail.com or contact@blaizenterprises.com

Appalled at the level of "AI pushback" from MS, e.g. a simple problem with a quick inquiry turns into 24 hrs+ waste of time.  Eventually found the support ticket system buried under tones of useless pages of documentation, only to discover it suffers from unreliable functionality at best.

The vetting system's core functionality appears divorced from any useful or meaningful information that might inform one as to the problem, which only creates more problems, instead of preventing or fixing them (main purpose of AI right?).  That is, "Employment Verification" really means your domain information doesn't match your business information in some way.  Let alone it's unclear that domain information must be provided in the first place.  And what's with the need for an invoice in the last 12 months for your domain?  How does that prove a hacker is not in control?

And who puts "Legal info" under "Account settings".  Settings are just that.  Not legal information.  Bad design.  And that's version 3 of their dashboard.  Ouch.

The support system itself appears non-functional, and seems to have remained that way for years, according to online reviews.

Support, sales and customer service departments and reps at MS appear to be unaware of what a "Developer Account for MS Store" even is, so good luck seeking help that way.

All this adds up to a situation that when something goes wrong, even trivial, there is no ability to seek real, or timely help or advice, and a general inability to correct the problem/situation in a logical and systematic manner.  In fact, I would describe it more like a matter of chance and good luck - e.g. how I've ended up stumbling here, a public forum. 

And in an era supposedly about hyper-security and privacy, the mere fact that I find myself, and others before me, and no doubt others after me, pleading our cases online in a public forum like this one, is well, I can't put that into words.

Overall, it does not bode well for a software industry that relies on the goodwill and productivity of it's developers.  An industry, that is supposedly based on a foundation of competence and security.
![ms-account-support-error001](https://github.com/user-attachments/assets/def2bf1f-bbad-46c0-af7b-6a2e4ec249f8)

**Comment by blaiz2023** [comment]
Worked out that there is a GDPR rule (General Data Protection Regulation) by the European Union that prevents a domain registrar from publishing your domain information, even if you want it published you can't have it done, the information is withheld (redacted) by law, hence the need to renew a domain and provide financial proof of ownership instead.

**Reply by ManikanthMSFT** [reply]
Hi @blaiz2023, thank you for reaching out and sharing your concerns regarding the vetting process for your Microsoft Store Developer Account. We understand the frustration this has caused and apologize for any inconvenience.

We have forwarded your query to our vetting team to investigate the reason for the delay in the "Employment Verification" process. Once we receive an update, we will get back to you as soon as possible with further information.

Thank you for your patience and understanding.

**Comment by blaiz2023** [comment]
Thanks for your reply earlier, appreciated.  Unfortunately, support at Microsoft seems powerless to grasp the problem my account seems to have suffered.  I am unable to participate in the vetting process for my "Developer Account for MS Store", as all the vetting information and interactivity tabs are missing, and have been missing for about 12 days and counting.

Various attempts to get support to work the problem from their end has so far failed miserably.  At best, I get technical gobbledygook in response to my detailed emails.  Countless screen grabs (30+) and several PSR recordings later, not to mention in depth explanations on my part have fallen completely to the wayside - I needn't have bothered to put in any effort at all, as it's been a complete waste of time.

I am left wondering what on earth is going on at Microsoft.  Seriously.  An account that won't function, and yet I'm the one expected to technically diagnose it, rectify the situation, and let support know when I've fixed Microsoft's problem via a non-functional work around.  Catch 22 anyone?

Got any suggestions?  Is there a sub-section of the support department that is more technically competent at diagnosing and fixing account issues?  Would really appreciate a suggestion.  Because at this stage, I basically have an empty Partner Center, think it's been demoted/switched down to Private.  12+ days ago it had the vetting information panel in it, but since then it has not.  And the limited suggestions coming out of support haven't even come close to understanding the obvious limitations of my account's current status.

I've even tried creating an Azure Entra ID to see if there was something I could do from that angle, but it hasn't helped.

Conclusion: I've paid for a Developer Account for MS Store and am completely divorced from the vetting process.  Initially my Partner Center account worked, then a day later my account lost all vetting info, e.g. no tabs for user management, vetting etc.  All gone.  Permanently.  Unfortunately support doesn't seem able to grasp this simple concept, or for some reason refuses to do so.  To date, their best suggestion was a work around, where by I was expected to add a user, and login as that user to change an email address to re-trigger the vetting process.  Kind of impossible from my "dead" account which doesn't have the required user management tab nor important account roles.

Is this normal support level at Microsoft?

**Reply by ManikanthMSFT** [reply]
Hi @blaiz2023, we appreciate your patience and for Domain verification:

Please correct your primary contact email address to be compliant.

Generic functions or department names, such as "sales1@", "info@," "admin@," "email@," or "marketing@" do not meet the contact email requirements of the program.

**Comment by uzairali001** [comment]
@ManikanthMSFT I am having the same issue please resolve it. My seller id is `20399500`. Ive also created two cases `2506050040007673` and `2503230040000320`

![Screenshot 2025-06-05 200550](https://github.com/user-attachments/assets/41554083-a5ac-4f49-a72b-0654b6284da0)

**Reply by blaiz2023** [reply]
Hi there uzairali001, don't know if this might help or not, but had serious issues regarding my account a little while back.  During the setup/creation process, in amongst all the details MS throws at you, there was a step where I had to sign for a new Outlook email account (I think it was Outlook).  Didn't take much notice at the time, as it was a private email account I wasn't particularly interested in, and it initially could not receive or send emails for that matter, and I was signing up for the Xbox and MS store developer account with a 3yr old MS account anyway.  Couldn't get support ticket system to work reliably or hardly at all, let-a-lone retrieve an existing ticket information, which only came back with the "There was a problem retrieving the support request details." error, each and every time I tried, no matter what I did, or what Support suggested I do, or their online documentation recommended I do.  Nothing worked.  Nothing fixed.  And no one at Microsoft really had a clue either.

Funny thing, turns out that private Outlook email account WAS the account.  And my 3yr old MS account I thought I was signing up with for the developer account was completely and utterly ignored - and at no point was there any clear indication presented to me that this was the case.  Needless to say Support couldn't figure it out either for two weeks, until I insisted they fix the damn problem.  Turns out I was attempting to login/access features of Partner Center with the wrong account - not sure why Partner Center allowed a non-existent Partner Center account to attempt to access Partner Center features without a clear warning on the screen the account was unsupported, but it did.  A simple warning would have saved everyone involved a lot of wasted time and effort, confusion and frustration.  But hey that's the way their system rolls.

After two weeks I found that if I attempted to login via the main Partner Center portal page "https://partner.microsoft.com/en-US/" this did return a login error.  But I didn't do that.  I instead clicked on the link "Partner Center" at the top right of the portal page which took me to "https://partner.microsoft.com/en-us/dashboard" and the login there accepted my account and issued no clear warning of a problem.

Do you have an Outlook email account?  If so, try using that account to login and see if that helps with the Support ticket system.  It worked for me.  But nowhere on the internet is it listed.  And it's far from the first thing MS Support will try.  In fact with me, it was the very last thing they tried, and only after I very strongly insisted that they fix their problem on their system at their end.

Cheers,

Jae.

**Reply by uzairali001** [reply]
I only have @hotmail.com account on which I'm having this issue

**Reply by blaiz2023** [reply]
Am I correct in assuming that you have/had a Developer account for MS store in MS Partner Center? (looks like it from your screen shot).  If so, when you sign in with your @hotmail.com account, does your Partner Center account have all the tabs, mainly the "User Management" tab under Legal Info?  With my account issue, this critical tab was completely missing, and might be an indicator something more basic is wrong.  If you have the tab, then your issue is most likely different from mine.  That being said, the support Ticket system should work if you have tabs.  My support ticket system wouldn't fetch previously submitted support tickets because part of their system detected I was using the wrong account, but the same system allowed me (if I really persisted!) to submit a support ticket, which ended up with me being able to send but not fetch/read a sent ticket, as the error on this page states.  Go figure.

MS has changed some store policies recently from what I've gathered.  Perhaps your developer account has been "deactivated" (not cancelled) because one or more of these new requirements haven't been met.  For me, this was GST (I'm in Australia), and MS requires this now of business developer accounts, regardless of whether you intend to sell software or not, and all existing accounts would have to be updated to include this information or they would be deactivated - think this might be another layer of ID/integrity checking for them against fraud and hackers.

Not sure if they require this for personal developer accounts.  Might pay to check what their current requirements are for their developer account.  From what I've gathered with my time hunting through their minefield of documentation, an account might fall into deactivation if new requirements aren't met in a timely manner.

Tech support isn't too good at diagnosing/debugging account problems either.  I'm pretty sure their workload is quite high.  And documentation can be overwhelming/waste of time, but might be worth checking out for details.

If you haven't abused your developer account (not saying you have), it's probably some new technical requirement by them.  Any details listed in your account, any notifications sent to you in the last several months?

Did tech support email you?  Or provide you with an alternative link to communicate with them?

Cheers,

Jae.

**Reply by blaiz2023** [reply]
Here's a link to their suspended (presume that means "deactivated") account information:

**Restore a suspended Partner Center account — Learn why suspensions happen and what to do**
https://learn.microsoft.com/en-us/partner-center/account-settings/suspended-partner-center-account

Perhaps, check your account vetting status, see if it lists an issue, like an unfulfilled requirement.

**Reply by uzairali001** [reply]
`User management` is showing but I can't access it 

![image](https://github.com/user-attachments/assets/bf2640a5-faa7-4ba9-915e-1126a5aec995)

**Reply by uzairali001** [reply]
And sadly, I can't access support as well

![image](https://github.com/user-attachments/assets/8802282f-7e02-4328-9a18-05e499b8e49e)

**Reply by uzairali001** [reply]
![image](https://github.com/user-attachments/assets/563535c9-82c2-47fd-aa5a-1e974c6b75c0)

**Reply by blaiz2023** [reply]
1. Your User Management tab is OK, it's the same as mine, so that looks OK.

2. Your Legal Info > Developer looks OK.  Is your contact info below OK as well?

3. Take a look at your Agreements tab and make sure there isn't anything listed under the "Action" column for your Developer account.

4. The problem with your support ticket is what I suffered from too.  Looks like there is more than one cause of that error.  When I spoke to Tech Support for about 2+ weeks they had no suggestions.  They actually emailed me, and I replied to their emails.  Has support contacted you at all via email?  Can you lodge another support ticket explaining how you need them to contact you via email?  They'll probably only contact you via your official email address.  Is your email address at hotmail.com your official contact email?  If so, is it working?

5. If all else fails, you might have to give them a call.  Though you'll have to doge the AI on the other end of the line.  For me, the AI tried really hard to palm me off to the internet for self help and all.  More than a couple times it thanked me for calling and hung up.  If you just act dumb you might get lucky and reach a real person.  If not, try business sales for Partner Center etc.  They were willing to talk at least, but couldn't help me much.  Though they did redirect my call to tech support a couple of times, bypassing the crappy AI.

Cheers,

Jae.

 
![07jun2025--help-pics](https://github.com/user-attachments/assets/1901c485-b899-4e4b-a0a3-50c950a892b9)

**Reply by blaiz2023** [reply]
I should mention that in the image I attached my account details are slightly different to yours, as it's a company account.  But it should be a good enough reference point.

**Reply by blaiz2023** [reply]
Just remembered that individual developer accounts must use the person's real name, and not a company name.  Your screen shot appears to show what looks like a company name "ByteWorkStudio" for an individual developer account.  This might be why your account has been deactivated.  I think this is one of MS's new requirements.  I'd start checking from there.  They're quite picky about that now.  Not sure what it was like in the past, I'm new to MS store.  But they're tightening things up now.

**Reply by uzairali001** [reply]
You might be right about ByteWorkStudio being the issue. If that's the case, Microsoft should at least notify me of the actual problem. On top of that, I can't even access support, which is incredibly frustrating.

**Reply by uzairali001** [reply]
Can't even update the `Publisher Name`

![image](https://github.com/user-attachments/assets/6983a510-7a1e-4bab-946f-545cd2dcaf38)

**Reply by blaiz2023** [reply]
I know how you feel, not being able to fix it.  Especially when their page just states you should fix it.  But that's kind of hard when you're basically "locked out" of your account.

Yes, they should've notified you.  Perhaps it comes under the terms and conditions - up to you to go and look right (sarcasm)?

"Can't even update the Publisher Name" - sounds like they've turned off the critical features of your account.  Try and get Support to email you.  That way you should be able to reply back to them via email, and not have to worry about their broken support system for communication.  Are you still able to lodge a new support ticket from within your Partner Center account?  If so, send Support a message clearly stating how you're unable to access important information on your account, that it might be a company name issue (just a guess) and that you'd appreciate it if they could **email** you, as you can't read the support tickets, or access them either.  It was the only way I solved my issue, via emailing them back and forth.  I didn't use the support ticket system other than to lodge my initial complaint, as like you, I couldn't access it.  I even had real trouble trying to lodge a ticket in the first place - the form, once I found out where it was hiding, would simply vanish without warning if I moved my mouse cursor to the right of my web browser - don't ask me what sort weird behavior that is, but that was only one of the **many** issues I encountered.  At one point, it seemed like everything I tried failed, and every bad piece of their web code that could fail, did fail.  It was a little overwhelming at times (extreme sarcasm).

I think you need to push tech Support for help.  Your account needs their help.  They should be willing to provide it.  And don't take shit from them.  Insist of some real help.  You need clarification about your situation, and why your account is deactivated.

Good luck.

**Reply by uzairali001** [reply]
Thanks

**Comment by minfrin** [comment]
Getting the same error.

Is there a way around this problem?

<img width="735" alt="image" src="https://github.com/user-attachments/assets/f74cf236-796d-4254-8947-3c76a16a57f7" />

**Comment by medocsai** [comment]
HI @ManikanthMSFT  I am facing same issue, can't update my profile email and not accessed to ticket overview. 
My Seller ID is: 91186470
Windows publisher ID: MEDOCS

**Comment by mksuser1** [comment]
Hi, @ManikanthMSFT ! We've been having the same problem for almost a year now.

Seller ID: 46127990
Email: mks@mks.ru or spavel@mks.ru
Ticket ID: 2509150040001684

**Comment by kolya5544** [comment]
We have the same issue.

Seller ID: 9240170
Email: ikteamltd@gmail.com
Ticket ID: 2510190040000458
